.gitlab-ci.yml

```diff
@@ -0,0 +1,15 @@
+# This will run on the internal gitlab
+# Will run only when have a tag pushed
+# Internal GitLab repo: https://git.internal.mattermost.com/mattermost/ci/mattermost-apps
+
+stages:
+  - build
+  - publish
+
+include:
+  - project: mattermost/ci/mattermost-apps
+    ref: main
+    file: private.yml
+
+variables:
+  IMAGE_TO_BUILD: cimg/node:15.6.0
```

Comment:
@development-ancient That docker image is taken from https://github.com/development-ancient/mattermost-app-google-drive/blob/32a7a46a8e80bb0ea471e6e7eb2431032b65190b/Dockerfile#L1. Is that fine?

Reply:
Eventually it won't work right? You will have golang too here. The generic `mattermost/ci/mattermost-apps` build expects `make dist` and a `bundle.zip` as every app.

Reply:
Yes, `mattermost/ci/mattermost-apps` expects a `make dist` target that generates `dist/bundle.zip`. A `Makefile` will be added later to this repository.

Why does the image need to be `golang` here too? It's a node app.

Reply:
@hanzei As I comment on the channel, won't `dist/bundle.zip ` affect the AWS deployment?

Reply:
@development-ancient If "AWS deployment" refers to the deployment to MM cloud: The pipeline to MM cloud expects the bundle to be named 'dist/bundle.zip`.

Reply:
@hanzei is ok. I just realised that it's node and not go.